src/Notes/AddFirstProduct.php

```diff
@@ -57,7 +57,8 @@ public static function get_note() {
 		}
 
 		$content_lines = array(
-			__( 'Nice one, you’ve created a WooCommerce store! Now it’s time to add your first product.<br/><br/>', 'woocommerce-admin' ),
+			'{greetings}<br/><br/>',
+			__( 'Nice one, you’ve created a WooCommerce store! Now it’s time to add your first product and get ready to start selling.<br/><br/>', 'woocommerce-admin' ),
```

Comment:
I know this was here prior to this PR, but I think the comma here is grammatically incorrect.  Either a terminal point or a semicolon may be more appropriate here.

Also, same comment about `sprintf` for `br` tags.